File: BEE 1074.py
n = int(input())
for i in range(n):
    num = int(input())
    if ((num % 2 == 0) and (num  > 0)):
        print("EVEN POSITIVE")
    elif ((num % 2 == 0) and (num  < 0)):
        print("EVEN NEGATIVE")
    elif ((num % 2 != 0) and (num  > 0)):
        print("ODD POSITIVE")
    elif ((num % 2 != 0) and (num  < 0)):
        print("ODD NEGATIVE")
    else:
        print("NULL")

File: BEE 1074.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var n = parseInt(lines.shift());

for(let i = 0; i < n; i++){
    var num = parseInt(lines.shift());
    if ((num % 2 === 0) && (num  > 0)){
        console.log("EVEN POSITIVE");
    }
    else if ((num % 2 === 0) && (num  < 0)){
        console.log("EVEN NEGATIVE");
    }
    else if ((num % 2 !== 0) && (num  > 0)){
        console.log("ODD POSITIVE");
    }
    else if ((num % 2 !== 0) && (num  < 0)){
        console.log("ODD NEGATIVE");
    }
    else{
        console.log("NULL");
    }
}